base_simulation.py

```diff
@@ -33,6 +33,31 @@ def calc_fkp(t):
         return 0.0
 
 
+def calc_kick_motion(t):
+    """Returns the lateral forced applied to the tire by the kick plate. The
+    force is modeled as a sinusoidal pulse."""
+    # TODO : return y, yd, ydd
+
+    start = 0.4  # seconds
+    stop = 0.6  # seconds
+    acc_magnitude = 50.0  # m/s/s
+
+    period = stop - start
+    frequency = 1.0/period
+    omega = 2*np.pi*frequency  # rad/s
+
+    magnitude = acc_magnitude/omega**2
+
+    if start + period/2 < t < stop:
+        y = magnitude/2.0*(1.0 - np.cos(omega*(t - start)))
+        yd = magnitude*omega*np.sin(omega*(t - start))/2
+        ydd = magnitude*omega**2*np.cos(omega*(start - t))/2
```

Comment:
This may need more careful thought. I take the same sinusoidal pulse we used for the force and differentiate twice. Then I set the magnitude to the peak desired acc.

Reply:
I think this isn't quite correct to end up with the desired pulse in acceleration. Needs checking.